### ISSUE TITLE
Should this be a valid ColumnSchema?

### ISSUE THREAD

**Issue by schnekli** [vmware/ovsdb-client-library#39]
```
{"type": "string", "ephemeral": true, "mutable": false}
```

column-schema is defined as
```
  <column-schema> := {
    "type": <type>                     required
    "ephemeral": <boolean>             optional
    "mutable": <boolean>               optional
  }
```
type is defined as
```
<type> - The type of a database column.  Either an <atomic-type> or a...
```
But when I do the following read and write in Groovy
```
objectMapper.readValue("""{"type": "string", "ephemeral": true, "mutable": false}""", ColumnSchema)
```
I get back
```
{"type":{"key":{}},"ephemeral":true,"mutable":false}
```

I think what _might_ be happening is that _StringBaseType_ is not serializing correctly.

**Comment by hechaoli** [comment]
Yes, this is because the serializer of `StringBaseType` is not implemented. This was kind of intentional because we are only interested in the deserialization of all *Schema object. The reason is that an OVSDB client NEVER sends a schema to the server. Only the server sends  a schema to the client through `get_schema` method. Thus IMHO, serialization of the schema objects are not necessary. Though you can still create a PR and fix it if you are interested.
